### PR TITLE
changes on landing and theme

### PR DESCRIPTION
**Pull request by practicatto**
there's still work to do but i'll do it on thursday or friday

accept my PR senpai >//<

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -88,36 +82,27 @@
...
       </p>
     </div>
-    <div class="my-8 mx-auto aspect-[4/5] w-40 bg-zinc-500" />
-  </div>
-</ChocoSection>
-<ChocoSection>
-  <SectionH2>Choco<wbr />misión</SectionH2>
-  <div class="flex flex-col gap-8 sm:grid sm:grid-cols-3">
-    {#each chocomisions as content, idx}
-      <Chocomision {content} {idx} />
```
Comment (adriandelgado) [inline src/routes/+page.svelte:105]: We don't use this component anymore. Let's delete the import

File: src/routes/Chocomision.svelte [hunk 2]
```diff
@@ -5,12 +5,3 @@
   const is_first = idx === 0;
   const current_num = (idx + 1).toString().padStart(2, "0");
 </script>
-
```
Comment (adriandelgado) [inline src/routes/Chocomision.svelte:8]: We can delete this component

**Review by adriandelgado — CHANGES_REQUESTED** [review]
Let's not forget `pnpm run check` and  `pnpm run lint`

File: src/routes/+page.svelte (hunk 1 above)
Comment (practicatto) [reply src/routes/+page.svelte:105]: done

**Review by adriandelgado — APPROVED** [review]
LGTM!

**Pull request merged**